### PR TITLE
Feature/1205276344521090 internal bridge rbac

### PR DESCRIPTION
**Pull request by talhadaar**
Creating this PR for discussion.

~TODO need to sync feature branch with parachain_dev after dependency updates.~

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205276344521090

### CODE REVIEW

File: precompiles/peaq-rbac/src/selectors.rs [hunk 1]
```diff
@@ -0,0 +1,46 @@
+use precompile_utils::keccak256;
+
+pub const SELECTOR_LOG_ADD_ROLE: [u8; 32] = keccak256!("RoleAdded(address,bytes32,bytes)");
```
Comment (neutrinoks) [inline precompiles/peaq-rbac/src/selectors.rs:3]: Seems to be the RBAC-ID-type, right? Especially for that, I defined that type within the peaq-primitives-xcm, have a look here: https://github.com/peaqnetwork/peaq-network-node/blob/185eff9e89db10221e38363f438ba761f61cbc7a/primitives/xcm/src/lib.rs#L87

**Review by neutrinoks — APPROVED** [review]
Ok from my side, but may you consider to replace the RBAC-ID-Type by the pre-defined datatype in our peaq-primitives-xcm. I wrote the URL on that one comment...

File: precompiles/peaq-rbac/src/selectors.rs (hunk 1 above)
Comment (talhadaar) [reply precompiles/peaq-rbac/src/selectors.rs:3]: Correct! But we're just hashing some string into a [u8;32] type, so I didn't think we needed to have a placeholder type.

**Review by sfffaaa — APPROVED** [review]

**Review by sfffaaa — COMMENTED** [review]
Please fix the `All checks have failed`

**Pull request merged**